TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -239,12 +242,12 @@ private static String createTweetText(Status status) {
     private static String createRelativeTime(Status status) {
         // Convert
         Date date = status.getCreatedAt();
-        String text = new TimeSpanConverter().toTimeSpanString(date);
+        String text = new TwitterTimeSpanConverter().toTimeSpanString(date);
 
         // Change text
-        text = text.replace("前", ""); //「〜分前」→「〜分」
-        text = text.replace("月", "/"); //「10月29日」→「10/29」
-        text = text.replace("日", "");
+//        text = text.replace("前", ""); //「〜分前」→「〜分」
+//        text = text.replace("月", "/"); //「10月29日」→「10/29」
+//        text = text.replace("日", "");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)